packages/sdks/src/functions/evaluate/node-runtime/node-runtime.ts

```diff
@@ -110,7 +111,7 @@ const getIvm = (): IsolatedVMImport => {
     
     SOLUTION: In a server-only execution path within your application, do one of the following:
   
-    ${SHOULD_MENTION_INITIALIZE_SCRIPT ? '- import and call `initializeNodeRuntime()` from "${SDK_NAME}/node/init".' : ''}
```

Comment:
oh no haha. did i really make this mistake...

thanks for catching it.

Reply:
i just thought my IDE was having trouble formatting it 😆